index.js

```diff
@@ -1,25 +1,49 @@
+/* eslint-disable no-var, prefer-template, comma-dangle */
+
 'use strict';
 
 module.exports = function buildAirbnbPreset(context, options) {
+  // This is similar to how `env` works in Babel:
+  // https://babeljs.io/docs/usage/babelrc/#env-option
+  // We are not using `env` because it’s ignored in versions > babel-core@6.10.4:
+  // https://github.com/babel/babel/issues/4539
+  // https://github.com/facebookincubator/create-react-app/issues/720
+  // It’s also nice that we can enforce `NODE_ENV` being specified.
```

Comment:
it seems this comment isn't valid anymore (note that i'm fine with requiring that it either be absent, or one of the three valid values, it's just that "absent" has to be OK)